index.html

```diff
@@ -12,18 +12,18 @@
     <main id="main">
         <div id="first">
```

Comment:
This is wonderful using id as a CSS selector and main as a semantic HTML element. I suggest you use the header where you have used the main so that they accurately describe the purpose of the element and the type of content that is inside them. example : <header class="header"><header>